### ISSUE TITLE
Site details constraints are breaking

### ISSUE THREAD

**Issue by SergioEstevao** [wordpress-mobile/WordPress-iOS#3822]
The iPad width constraint is breaking on iPad2 8.1 simulator.  Layout itself doesn't seem to be affected.

``` 
WordPress[24888:10001163] Unable to simultaneously satisfy constraints.
    Probably at least one of the constraints in the following list is one you don't want. Try this: (1) look at each constraint and try to figure out which you don't expect; (2) find the code that added the unwanted constraint or constraints and fix it. (Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSLayoutConstraint:0x7d0b9120 H:[UIView:0x7be403a0]-(6)-|   (Names: '|':UIView:0x7be4e760 )>",
    "<NSLayoutConstraint:0x7d0784c0 H:|-(6)-[UIView:0x7be403a0]   (Names: '|':UIView:0x7be4e760 )>",
    "<NSLayoutConstraint:0x7bebc700 H:[UIView:0x7be4e760]-(0)-|   (Names: '|':UITableViewCellContentView:0x7d0061b0 )>",
    "<NSLayoutConstraint:0x7bebc730 H:|-(0)-[UIView:0x7be4e760]   (Names: '|':UITableViewCellContentView:0x7d0061b0 )>",
    "<NSLayoutConstraint:0x7be596b0 H:[UIView:0x7be403a0(600)]>",
    "<NSLayoutConstraint:0x7b960f80 'UIView-Encapsulated-Layout-Width' H:[UITableViewCellContentView:0x7d0061b0(320)]>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x7be596b0 H:[UIView:0x7be403a0(600)]>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful. 
```

**Comment by aerych** [comment]
@SergioEstevao Can you double check the error?  That looks like the same one addressed in #3817. This is what I'm seeing on the site details screen: 

```
2015-06-09 10:15:17.666 WordPress[2471:15913] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. Try this: (1) look at each constraint and try to figure out which you don't expect; (2) find the code that added the unwanted constraint or constraints and fix it. (Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSLayoutConstraint:0x7bf39770 UIView:0x7bea8790.leading == UITableView:0x7a9fc000.leadingMargin>",
    "<NSAutoresizingMaskLayoutConstraint:0x7ba4b9e0 h=--& v=--& UIView:0x7bea8790.midX == + 512>",
    "<NSAutoresizingMaskLayoutConstraint:0x7ba897f0 h=--& v=--& H:[UIView:0x7bea8790(1024)]>",
    "<NSAutoresizingMaskLayoutConstraint:0x7bab6620 h=-&- v=-&- 'UIView-Encapsulated-Layout-Left' H:|-(0)-[UITableView:0x7a9fc000]   (Names: '|':_UIParallaxDimmingView:0x7bed7cd0 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x7bf39770 UIView:0x7bea8790.leading == UITableView:0x7a9fc000.leadingMargin>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
```

When checking I also noticed that there are breaking constraints in the table header and footer views for the post and page list.  I'll look at addressing those in a different issue.

**Comment by SergioEstevao** [comment]
@aerych @sendhil The error I'm seeing is this: ```Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. Try this: (1) look at each constraint and try to figure out which you don't expect; (2) find the code that added the unwanted constraint or constraints and fix it. (Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSLayoutConstraint:0x7978f260 UIView:0x7c14b6c0.leading == UITableView:0x7b4c3c00.leadingMargin>",
    "<NSAutoresizingMaskLayoutConstraint:0x7a6f15d0 h=--& v=--& UIView:0x7c14b6c0.midX == + 384>",
    "<NSAutoresizingMaskLayoutConstraint:0x7a6f1260 h=--& v=--& H:[UIView:0x7c14b6c0(768)]>",
    "<NSAutoresizingMaskLayoutConstraint:0x7a6b7830 h=-&- v=-&- 'UIView-Encapsulated-Layout-Left' H:|-(0)-[UITableView:0x7b4c3c00]   (Names: '|':_UIParallaxDimmingView:0x7a0ba750 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x7978f260 UIView:0x7c14b6c0.leading == UITableView:0x7b4c3c00.leadingMargin>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.```
